draft-bradley-dnssd-private-discovery.md

```diff
@@ -291,9 +305,7 @@ The network infrastructure may leak identifiers in the form of persistent IP add
 The following are some of the things that still need to be specified and decided:
 
 * Figure out how sleep proxies might work with this protocol.
-* Define probe and announcement random delays to reduce collisions.
 * Describe when to use the same EPK2 in a response to reduce churn on probe/response collisions.
-* Consider randomly answering probes for non-friends to mask real friends.
```

Comment:
This topic also needs more work. I was originally thinking of a whole section on fake answers. But I guess that could be a separate document since it's a big discussion.